site/src/layouts/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from "./DetailComponent";
```

Comment:
I would rename the Layouts and remove this Detail prefix (needs Mosaic change)

Reply:
I'll make the change after we change it in Mosaic :)